package.json

```diff
@@ -58,13 +58,13 @@
     "minimist": "^1.1.0",
     "mkdirp": "~0.3.5",
     "moment": "^2.6.0",
-    "mongoose": "~3.8.21",
-    "mongoose-long": "^0.0.2",
+    "mongoose": "^4.13.0",
+    "mongoose-long": "^0.1.1",
     "node-minify": "^1.2.1",
     "parsimmon": "^0.4.0",
     "passport": "^0.2.0",
     "passport-local": "^1.0.0",
-    "q": "1.0.1",
+    "q": "^1.5.1",
```

Comment:
The same here.

Reply:
You're right, I've changed to only minor versions `~`